include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -1409,10 +1409,22 @@ make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
 
             void write(ThreadParams* threadParams, std::string mpiTransportParams)
             {
+                const SubGrid<simDim>& subGrid = Environment<simDim>::get().SubGrid();
                 /* y direction can be negative for first gpu */
```

Comment:
Is this comment needed? I think in the new code it's already expressed in line 1420

Reply:
Yes  I will remove the comment.